projects.csv

```diff
@@ -1,2 +1,2 @@
-name,path
-jsoup,https://github.com/jhy/jsoup
\ No newline at end of file
+path,name
+https://github.com/jhy/jsoup
```

Comment:
no name?

Reply:
@rafaelmotaalves it's better to have two projects in the example. one with name and the other without name